### PR TITLE
Highlight series and corresponding legend

### PR DESCRIPTION
**Pull request by joesis**
Two improvements:
1. Legend doesn't hide the graph of interest any more.
1. The trend can be checked more apparently when hovering on the series.

zeno

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -42,7 +42,7 @@
   name = "github.com/codahale/hdrhistogram"
 
 [[constraint]]
-  branch = "master"
+  version = "~1.1.0"
   name = "github.com/davecgh/go-spew"
```
Comment (joesis) [inline Gopkg.toml]: This solves the conflict with `testify` package found by `dep ensure`.

File: web/index.go [hunk 2]
```diff
@@ -504,8 +516,16 @@ var indexHTML = []byte(`
         data,
         {
           labels: labels,
+          labelsSeparateLines: true,
+          labelsKMG2: true,
```
Comment (joesis) [inline web/index.go]: Doc at http://dygraphs.com/options.html#labelsKMG2

**Comment by joesis** [comment]
CI kept hanging because of seemingly unrelated reason.

File: web/index.go [hunk 3]
```diff
@@ -504,8 +516,16 @@ var indexHTML = []byte(`
...
           includeZero: true,
-          legend: 'always',
+          legend: 'follow',
```
Comment (joesis) [inline web/index.go]: One downside of this change is that the legend won't show by default on ops panel. It causes no trouble to me as the first reaction to unusual graphs is hovering to see the actual data. A custom `legendFormatter` can be used if always showing legend is a requirement.

**Comment by oxtoacart** [comment]
Nice!

**Comment by joesis** [comment]
Thanks @oxtoacart !

**Pull request merged**